### ISSUE TITLE
Hyperparameters to reproduce the results

### ISSUE THREAD

**Issue by ukyh** [SleepThroughDifficulties/KernelWhitening#2]
Hi, thank you for sharing the excellent code!
I ran `mnli_run.sh` with multiple seeds but found the results did not match the ones of the paper.
I wonder if the script differs from the one used in the paper.
If so, could you share the scripts (or just hyperparameters) to reproduce the results of the paper, including QQP and FEVER?
Thank you in advance!

**Comment by SleepThroughDifficulties** [comment]
Thanks for your interest in our work.
I'm sorry that I have changed my computer and lost the original script file.
I think the default hyperparameters in mnli_run.sh are the correct setting for experiments. Maybe you can try to use a larger learning rate or an early exit strategy or change the lrbr for better performance. 
The results do vary a lot, I recommend researching something else as LLM seems automatically perform better on spurious correlations. Maybe its a hard question for Bert only.
Best
